Examples/Tests/Macroscopic_Maxwell/inputs_3d.evolveM

```diff
@@ -0,0 +1,92 @@
+# ###############################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 1000
+amr.n_cell = 8 8 8 # number of cells spanning the domain in each coordinate direction at level 0
+amr.max_grid_size = 512 # maximum size of each AMReX box, used to decompose the domain
+amr.blocking_factor = 8
+geometry.coord_sys   = 0
+geometry.is_periodic = 1  1  1
+geometry.prob_lo     =  -1.5e-6 -1.5e-6 -1.5e-6
+geometry.prob_hi     =   1.5e-6  1.5e-6  1.5e-6
+amr.max_level = 0
+
+#################################
+############ NUMERICS ###########
+#################################
+warpx.verbose = 0
+warpx.use_filter = 0
+warpx.cfl = 4000
```

Comment:
just out of curiosity - this cfl > 1 because of the choice of alpha that you mentioned? is the choice of cfl and alpha related.?

Reply:
nope. CFL has nothing to do with alpha... I figured shortly after our meeting that with only LLG, stability is not an issue, so we can choose as large as dt we want (to satisfy Nyquist rule of course)